src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -52,8 +57,6 @@ public static class ConnectionBuilder
     /// </summary>
     public class ConnectionBuilder<TSourceType>
     {
-        private bool _isUnidirectional;
-
         private bool _isBidirectional;
 
         private int? _pageSize;
```

Comment:
```suggestion
        internal const string _bidirectionalMetadataKey = "__ConnectionBuilder_Bidirectional";
        internal const string _pageSizeMetadataKey = "__ConnectionBuilder_PageSize";

        private bool _isBidirectional
        {
            get => FieldType.GetMetadata<bool>(_bidirectionalMetadataKey);
            set => FieldType.WithMetadata(_bidirectionalMetadataKey, value.Boxed());
        }

        private int? _pageSize
        {
            get => FieldType.GetMetadata<int?>(_pageSizeMetadataKey);
            set => FieldType.WithMetadata(_pageSizeMetadataKey, value);
        }
```

Reply:
?

Reply:
Only for page size. Why internal ?

Reply:
Bidirectional or Unidirectional can be calculated from the presence or absence of required arguments.

Reply:
`internal` so that it can be accessed from `ConnectionBuilder<TSourceType, TReturnType>`.  In v5 we will remove this class altogether of course, and then the const can be private.

`_bidirectional` can be inferred as you say, like this I assume:

```cs
private bool _isBidirectional => FieldType.Arguments.Any(x => x.Name == "before");
```

Reply:
Do you think the const should be public?

Reply:
Please review updated code.

Reply:
Internal is ok for now.